lib/output/blob_storage.go

```diff
@@ -19,13 +19,16 @@ Sends message parts as objects to an Azure Blob Storage Account container. Each
 object is uploaded with the filename specified with the ` + "`container`" + `
 field.`,
 		Description: `
+The output can be authenticated through a connection string or a storage account name with 
+the respective access key. If both authentication methods are set, the connection string is used.
 In order to have a different path for each object you should use function
 interpolations described [here](/docs/configuration/interpolation#bloblang-queries), which are
 calculated per message of a batch.`,
 		Async: true,
 		FieldSpecs: docs.FieldSpecs{
 			docs.FieldCommon("storage_account", "The storage account to upload messages to."),
 			docs.FieldCommon("storage_access_key", "The storage account access key."),
+			docs.FieldCommon("storage_connection_string", "The storage account connection string."),
```

Comment:
This description ought to specify that only one of `storage_connection_string` or `storage_account` + `storage_access_key` is required.

Reply:
I put it at the beginning of the output description, but perhaps it's not clear enough, I'll try to rephrase it.

Reply:
I think the wording further up is fine, but I'd expect users to skip straight to the field docs quite often so it's probably best there instead.

Reply:
I add it now to the field docs